### PR TITLE
Add project Hypercubed/angular-marked

### PR DESCRIPTION
**Pull request by ScottRFrost**

### CODE REVIEW

**Comment by ScottRFrost** [comment]
Forgot a " .  Sorry about that!

File: files/angular-marked/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "angular-marked",
+  "repo": "Hypercubed/angular-marked",
+  "files": {
+    "include": ["angular-marked.js", "angular-marked.min.js"]
```
Comment (megawac) [inline files/angular-marked/update.json]: need `basePath: 'dist/'`?

**Comment by ScottRFrost** [comment]
Woops, I thought the file search was recursive.  Added basePath.

File: files/angular-marked/update.json [hunk 2]
```diff
@@ -2,6 +2,7 @@
   "packageManager": "github",
   "name": "angular-marked",
   "repo": "Hypercubed/angular-marked",
+  "basePath": "dist/",
```
Comment (megawac) [inline files/angular-marked/update.json]: basePath goes in `files`

**Comment by ScottRFrost** [comment]
Fixed it.  Thanks.  The documentation around basePath isn't super clear.

**Pull request merged**